tests/sync/session/session.cpp

```diff
@@ -356,6 +361,7 @@ TEMPLATE_TEST_CASE("sync: stop policy behavior", "[sync]", RegularUser) {
     if (!EventLoop::has_implementation())
         return;
 
+    SyncManager::shared().reset_for_testing();
```

Comment:
Nothing should have to ever call reset_for_testing() at the start of the test. The previous test should be cleaning up properly.